automatic/git.install/tools/helpers.ps1

```diff
@@ -0,0 +1,64 @@
+function Get-InstallKey() {
+    $registryKeyName = 'Git_is1'
+    $installKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+    if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -eq 'true') {
```

Comment:
There used to only be a 32 bit version. The logic here can likely be refactored/removed entirely